fe/fe-core/src/main/java/com/starrocks/sql/common/TimeUnitUtils.java

```diff
@@ -0,0 +1,40 @@
+// Copyright 2021-present StarRocks, Inc. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.starrocks.sql.common;
+
+import com.google.common.collect.ImmutableMap;
+
+public class TimeUnitUtils {
+
+    public static final String SECOND = "second";
```

Comment:
why not use enum?

Reply:
This PR only splits  the code into an independent file Which havn't change the codes yet.